### PR TITLE
Fix clip tests

### PR DESCRIPTION
**Pull request by AlexKordic**
Removed failing checks.

Didn't found good way to replace temporary failing checks.

Used manual test to verify proper operation:
- got input file timestamps: [frames_input.csv](https://github.com/livepeer/lpms/files/8397619/frames_input.csv)
- got encoded file timestamps: [frames_output.csv](https://github.com/livepeer/lpms/files/8397620/frames_output.csv)
- Produced video track contains proper frames
- Verified frames visually

Hard to implement above checks in code without additional image transformations. @oscar-davids any methods available we can use to verify correct set of frames are in output?

# Flushing procedure fix

When using tensorflow and flushing output in `process_out()` function `getmetadatainf()` call causes error to propagate and `TestTranscoder_DetectionFreq` test fails.

Added [condition](https://github.com/livepeer/lpms/blob/d309a54bcd234a5c7369a1e720c2c0d2d957ecc6/ffmpeg/encoder.c#L532) to cover this case.

### CODE REVIEW

File: doc/trim_output_tracks.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+```
+V V V V V
+AAAAAAAAAA
+
+output:
+V V V 
+AAAAAA
+```
+
+offsted input:
```
Comment (thomshutt) [inline doc/trim_output_tracks.md:35]: offset?

File: doc/trim_output_tracks.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```
+    V V V V V
+AAAAAAAAAA   
+
+output:
+V V V
+ AAA
```
Comment (thomshutt) [inline doc/trim_output_tracks.md:52]: How come we don't get the audio frame that lines up with the first video frame in the output?

**Review by thomshutt — COMMENTED** [review]
Looks fine to me - if @oscar-davids and @cyberj0g don't have any thoughts on improving the stability of these bits of the tests then I'd rather get rid of them and be able to rely on the test suite again

File: doc/trim_output_tracks.md (hunk 1 above)
Comment (AlexKordic) [reply doc/trim_output_tracks.md:35]: There are 3 cases: both audio and video start on same timestamp or audio can start first or video can start first. In each case starting timestamp is not required to be 0 (common in splicing streams)

**Review by cyberj0g — APPROVED** [review]

**Pull request merged**